lib/cap.c

```diff
@@ -40,6 +40,8 @@
 #include "libvfio-user.h"
 #include "cap.h"
 
+#define VFU_MAX_CAPS 48
```

Comment:
Why  change the original way we calculated it? `(PCI_CFG_SPACE_SIZE - PCI_STD_HEADER_SIZEOF) / PCI_CAP_SIZEOF`

Reply:
It needs to go away anyway as it's not right, but yeah, I should restore that.